tests/test_livingspace.py

```diff
@@ -2,11 +2,19 @@
 import unittest  
 
 
-class Test(unittest.TestCase):
+class TestLivingspace(unittest.TestCase):
 	
 	def setUp(self):
 		self.livingspace = LivingSpace()
 
 	def test_livingspace_is_not_full(self):
 		result = self.livingspace.isFull()
-		self.assertEqual(result,False, msg = "The livingspace is now full")	 
\ No newline at end of file
+		self.assertEqual(result,False)	 
+
+	def test_livingspace_is_full(self):
+		self.livingspace.current_members = ["marvin","james","kang","andrew","austin"]
```

Comment:
This test seems very obvious too. It doesn't use a function to add people to the room. Its similar to saying x=['a', 'b','c','d'] and checking if len is 4. Your test is not expecting, its just asserting the obvious.